app/assets/javascripts/components/story/StoryAttachment.js

```diff
@@ -78,6 +118,7 @@ class StoryAttachment extends React.Component {
         data-attachinary={attachinary.dataAttachinary}
         data-form-data={attachinary.dataFormData}
         data-url={attachinary.dataUrl}
+        onClick={() => { this.updateSignature() }}
```

Comment:
Do we need some "lock"/"wait" in the submit action to wait for this request to complete?